recipes/pypgatk/meta.yaml

```diff
@@ -0,0 +1,115 @@
+{% set name = "pypgatk" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c249729dcd9c344f88469826a7d023cd4ada3236172832fa90a950208b8576da"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - argcomplete ==1.9.5
```

Comment:
You also most likely do only need python and pip here in the host section.

Reply:
I generate this with the skeleton option of conda and did some refinements. I will remove other dependencies. Is that the same for requirements run?

Reply:
Keep everything in the "run:" section since they're needed for the package to actually be used.

Reply:
@dpryan79 changes added!!!! Thanks a lot for the feedback.